xml/libvirt_guest_installation.xml

```diff
@@ -565,10 +567,145 @@ network=vnet_nated</screen>
+        is required, <emphasis>nvram</emphasis> defines an optional UEFI
+        variable store. The following example illustrates manual firmware
+        selection with secure boot enabled.
+      </para>
+
+<screen>&prompt.user;virt-install --connect qemu:///system --virt-type kvm \
+--name sle15sp5 --memory 1024 --disk size=10 --cdrom /dev/cdrom --graphics vnc \
+--boot loader=/usr/share/qemu/ovmf-x86_64-smm-code.bin,loader.readonly=yes,loader.type=pflash,loader.secure=yes,nvram.template=/usr/share/qemu/ovmf-x86_64-smm-vars.bin \
+--os-variant sle15sp5</screen>
+
+      <note>
+        <para>
+          &libvirt; cannot modify any characteristics of the UEFI firmwares. For
+          example it cannot disable &uefisecboot; in a firmware that has
+          &uefisecboot; enabled, even when specifying <emphasis>loader.secure=no</emphasis>.
+          &libvirt; will ensure the specified firmware is capable of satisfying
+          any specified features. For example it will reject configuration that
+          disables secure boot with <emphasis>loader.secure=no</emphasis>, but
+          specifies a firmware that has &uefisecboot; enabled.
+        </para>
+      </note>
+
+      <para>
+        The <package>qemu-ovmf-x86_64</package> package contains several UEFI
+        firmware images. For example the following subset supports SMM,
+        &uefisecboot; and have either Microsoft, openSUSE, or SUSE UEFI CA keys
+        enrolled:
+      </para>
+
+<screen>
+&prompt.root;<command>rpm -ql qemu-ovmf-x86_64</command>
+[...]
+/usr/share/qemu/ovmf-x86_64-smm-ms-code.bin
+/usr/share/qemu/ovmf-x86_64-smm-ms-vars.bin
+/usr/share/qemu/ovmf-x86_64-smm-opensuse-code.bin
+/usr/share/qemu/ovmf-x86_64-smm-opensuse-vars.bin
+/usr/share/qemu/ovmf-x86_64-smm-suse-code.bin
+/usr/share/qemu/ovmf-x86_64-smm-suse-vars.bin
+[...]
+</screen>
+
+      <para>
+        For the &aarch64; architecture, the package is named
+        <package>qemu-uefi-aarch32</package>:
+      </para>
+
+<screen>
+&prompt.root;<command>rpm -ql qemu-uefi-aarch32</command>
+[...]
+/usr/share/qemu/aavmf-aarch32-code.bin
+/usr/share/qemu/aavmf-aarch32-vars.bin
+/usr/share/qemu/firmware
+/usr/share/qemu/firmware/60-aavmf-aarch32.json
+/usr/share/qemu/qemu-uefi-aarch32.bin
+</screen>
+
+      <para>
+        The <filename>*-code.bin</filename> files are the UEFI firmware files.
+        The <filename>*-vars.bin</filename> files are corresponding variable
+        store images that can be used as a template for a per-VM non-volatile
+        store. &libvirt; copies the specified <literal>vars</literal> template
+        to a per-VM path under <filename>/var/lib/libvirt/qemu/nvram/</filename>
+        when first creating the VM. Files without <literal>code</literal> or
+        <literal>vars</literal> in the name can be used as a single UEFI image.
+        They are not as useful since no UEFI variables persist across power
```

Comment:
```suggestion
        They are not as useful since no UEFI variables persist across the power
```

Reply:
I have not applied this suggestion. It's not needed IMO and makes the sentence a bit clumsier.